src/index.ts

```diff
@@ -1,13 +1,19 @@
 import 'dotenv/config';
 
 import app from './server';
+import startDB from './db/db_config';
 
 const PORT = process.env.PORT || 3001;
 
-const startApp = () => {
-  app.listen(PORT, () => {
-    console.log(`App has been started on the port ${PORT}`);
-  });
+const startApp = async () => {
+  try {
+    await startDB();
```

Comment:
this logic better to move to server file

Reply:
Do you mean that startApp function declaration should be moved to server file and in the index this function should be just called?